src/Form/SettingsForm.php

```diff
@@ -45,25 +53,29 @@ final class SettingsForm extends ConfigFormBase {
    *   The entity type manager.
    * @param \Drupal\azure_ad_delta_sync\UserManagerInterface $userManager
    *   The user manager.
+   * @param \Drupal\azure_ad_delta_sync\Helpers\ConfigHelper $configHelper
+   *   The configuration helper.
    *
    * @throws \Drupal\Component\Plugin\Exception\InvalidPluginDefinitionException
    * @throws \Drupal\Component\Plugin\Exception\PluginNotFoundException
    */
-  public function __construct(ConfigFactoryInterface $configFactory, EntityTypeManager $entityTypeManager, UserManagerInterface $userManager) {
+  public function __construct(ConfigFactoryInterface $configFactory, EntityTypeManager $entityTypeManager, UserManagerInterface $userManager, ConfigHelper $configHelper) {
```

Comment:
You can use https://www.php.net/manual/en/language.oop5.decon.php#language.oop5.decon.constructor.promotion to define `$configHelper`:

```suggestion
  public function __construct(ConfigFactoryInterface $configFactory, EntityTypeManager $entityTypeManager, UserManagerInterface $userManager, private ConfigHelper $configHelper) {
```

(and remove the declaration in line 45 and also remove line 67).